livre/core/types.h

```diff
@@ -237,6 +237,7 @@ const uint32_t NODEID_FRAME_BITS = 18; //>! @see NodeId
 const uint32_t INVALID_POSITION = ( 1u << NODEID_BLOCK_BITS ) - 1; //!< Invalid node ID.
 const uint32_t INVALID_LEVEL = ( 1u << NODEID_LEVEL_BITS ) - 1; //!< Invalid tree level.4 bits is on
 const uint32_t INVALID_FRAME = ( 1u << NODEID_FRAME_BITS ) - 1; //!< Invalid tree level.4 bits is on
+const uint32_t LATEST_FRAME = INT_MAX;
```

Comment:
I tried LONG_MAX and ULONG_MAX, but it didn't work, as we are comparing that value with params.animation, which is signed (it can be negative for backwards playback).